v2/manager.go

```diff
@@ -245,12 +251,17 @@ func (c *Manager) Procs(recursive bool) ([]uint64, error) {
 	return processes, err
 }
 
-func (c *Manager) Stat() (map[string]uint64, error) {
+var singleValueFiles = []string{
+	"pids.current",
+	"pids.max",
```

Comment:
can be another PR, but we need `memory.current` and `memory.swap.current`?
https://facebookmicrosites.github.io/cgroup2/docs/memory-controller.html